src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -2900,6 +2900,23 @@ void M()
             await VerifyItemExistsAsync(markup, "myClass1", glyph: (int)Glyph.Local, options: options);
         }
 
+        [WorkItem(61747, "https://github.com/dotnet/roslyn/issues/61747")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task NotAfterEventName()
```

Comment:
❗ Also need to add a test that the following still works:

```
class Test { }

public class MyClass
{
    private event System.EventHandler Test;
    void M()
    {
        Test $$
    }
}
```

Reply:
If to fix the bug with only syntax checks, then this test will produce `eventHandler`, `@event` and `handler` suggestions instead of `test`. I don't see a syntax-only way to overcome it.